DeezyMatch/candidateRanker.py

```diff
@@ -245,15 +263,22 @@ def candidate_ranker(input_file_path="default", query_scenario=None, candidate_s
 
     # Empty dataframe to collect data
     output_pd = pd.DataFrame()
+    
     for iq in range(len_vecs_query):
         print("=========== Start the search for %s" % iq, vecs_items_query[iq][1])
         collect_neigh_pd = pd.DataFrame()
         num_found_candidates = 0
-        # start with 0:seach_size
+        
+        # start with 0:search_size
         # If the number of selected candidates < num_candidates
         # Increase the search size
         id_0_neigh = 0
         id_1_neigh = search_size
+
+        # If use_predict is false, the search strategy is skipped
+        if use_predict == False:
+            id_1_neigh = search_size
```

Comment:
@mcollardanuy What is the difference between this and line 276? `id_1_neigh = search_size`? Should it be `len(vecs_candidates)`?

Reply:
Oh, I think this if condition may be from an old version of the code!

Reply:
But the if condition is about the new arg: `use_predict`? (Sorry maybe I misunderstood)

Reply:
Sorry, what I meant is that I think I added it in the first commit and then forgot to remove it. In any case, I think we don't need this condition at this point. Do you think this makes sense?

Reply:
👍 :